Formula/mercurial.rb

```diff
@@ -13,6 +13,13 @@ class Mercurial < Formula
     sha256 "8934a8fc26c36e56f017d99ac43a0ebe9d6922f8276c1dfccd9ccb980ae05e45" => :mavericks
   end
 
+  if MacOS.version <= :snow_leopard
+    depends_on :python
```

Comment:
Could we follow this pattern instead: https://github.com/Homebrew/homebrew-core/blob/c91fecf77a056011802d81582952d2e9623838d4/Formula/macvim.rb#L27-L31 for MacVim where we're doing a similar thing?